evo_researcher/autonolas/research.py

```diff
@@ -486,7 +487,7 @@ def get_urls_from_queries(
     return list(results)
 
 
-def standardize_date(date_text):
+def standardize_date(date_text: str) -> str | None:
```

Comment:
`typing.Optional` is used elsewhere in this repo, so we should use e.g. `typing.Optional[str]` here, and same for other cases in this PR

Reply:
It's a feature of a supported version of Python in this repo, does it matter which one is used?

Reply:
It could have sounded harsh, so just to enhance my answer a bit 😄 

I'd say if some code style can be enforced (black, flake, ruff, isort, ...) then let's agree on the tool and let it do the checking. If some tool isn't checking for it, it will get out of sync anyway.

Reply:
Fair enough!